r/tests/testthat/test-duckdb.R

```diff
@@ -15,6 +15,15 @@
 # specific language governing permissions and limitations
 # under the License.
 
+test_that("meaningful error message when duckdb is not installed", {
```

Comment:
Could we move the `skip_on_cran()` and skip_if_not_available("dataset") to be above this test? We need datasets for the test to run, and it might be paranoid, but want to skip even this test on cran just in case

Reply:
Done